### ISSUE TITLE
Adjust layout of event listings

### ISSUE THREAD

**Issue by danieltott** [Virtual-Coffee/virtualcoffee.io#62]
## Issue Context

In order to improve scan-ability, let's move a couple things around. 

## Suggested solution:

- Move date up into a `.card-header` and make it a link
- Make the last link into a `.btn`

The output will end up looking like this:

```html
<div class="card mb-4">
  <a href="https://meetingplace.io/virtual-coffee/events/3183" class="card-header" target="_blank" rel="noopener noreferrer">October 20, 2020, 9:00 AM EDT</a>
  <div class="card-body">
    <h5 class="card-title">Virtual Coffee - Morning Crowd</h5>
    <p class="card-text">Description if it exists</p>
    <a href="https://meetingplace.io/virtual-coffee/events/3183" class="btn-sm btn btn-secondary" target="_blank" rel="noopener noreferrer">Details and RSVP</a>    
  </div>
</div>
```

I'm pulling these classes from [the bootstrap card docs](https://getbootstrap.com/docs/4.5/components/card/#header-and-footer)

**Comment by andreamartz** [comment]
Hi Dan.
I'll tackle this one with Courtney.
Andrea

**Comment by danieltott** [comment]
Closed in #67
Thank you @andreamartz - great work on these!!

![](https://media3.giphy.com/media/nj0wszRab0Mxi/giphy.gif)
